### PR TITLE
M1 Compatibility

### PR DESCRIPTION
**Pull request by doovers**
### Description
- Updates CocoaLumberjack to 3.7.2
- Resolves some deprecation warnings
- Bumps version to 1.0.2

### Change Request:
https://life360.atlassian.net/browse/SYSENG-7634

### CODE REVIEW

File: MQTTClient.podspec [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-		:tag => "1.0.1",
+		:tag => "1.0.2",
 		:submodules => true
 	}
 
 	mqttc.requires_arc = true
-	mqttc.platform = :ios, "6.1", :osx, "10.10", :tvos, "9.0"
-	mqttc.ios.deployment_target = "6.1"
+	mqttc.platform = :ios, "14.0", :osx, "10.10", :tvos, "9.0"
+	mqttc.ios.deployment_target = "14.0"
```
Comment (Vkt0r) [inline MQTTClient.podspec:16]: Are we not still supporting iOS 13 ??

Comment (doovers) [reply MQTTClient.podspec:16]: No I don't think so. `iphone` develop branch has 14

Comment (Vkt0r) [reply MQTTClient.podspec:16]: Really ?? Uhmm all our Kits and Engine are still in iOS 13. Yes you're right

```
IPHONEOS_DEPLOYMENT_TARGET = 14.0
```

We are good then!

Comment (doovers) [reply MQTTClient.podspec:16]: FYI I'm in the process of updating some of our kits/engines too as part of the genesis work

**Pull request merged**